components/PlantCard.js

```diff
@@ -6,56 +6,62 @@ import Image from "next/image";
 
 export default function PlantCard({ plant, toggleFavourite }) {
   return (
-    <Container>
-      <button
+    <StyledArticle>
+      <ButtonFavourite
         aria-label={
           plant.isFavourite ? "Remove from favorites" : "Add to favorites"
         }
         onClick={() => toggleFavourite(plant.id)}
       >
         {plant.isFavourite ? <PlantFavIconFill /> : <PlantFavIcon />}
-      </button>
+      </ButtonFavourite>
       <h2>{plant.name}</h2>
-      <StyledParagraph>{plant.botanicalName}</StyledParagraph>
+      <p>{plant.botanicalName}</p>
 
-      <RoundImage
+      <Image
         alt={`Image of ${plant.name}`}
         width={200}
         height={200}
         src={plant.imageUrl || "/assets/empty.avif"}
       />
 
-      <StyledLink href={`/${plant.id}`}>see more</StyledLink>
-    </Container>
+      <SeeMoreLink href={`/${plant.id}`}>see more</SeeMoreLink>
+    </StyledArticle>
   );
 }
 
-const StyledLink = styled(Link)`
-  border: 1px solid black;
-  border-radius: 8px;
-  background-color: lightgray;
-  color: black;
-  text-decoration: none;
-  padding: 5px;
+const StyledArticle = styled.article`
+  padding: 10px 10px 30px;
+  margin: 0 21px 23px -20px;
 `;
 
-const RoundImage = styled(Image)`
-  border-radius: 100px;
-`;
+const ButtonFavourite = styled.button`
+  position: absolute;
```

Comment:
We have an `article` defined in `styles.js` with `position: absolute`. We use StyledArticle here because it requires specific positioning. The same goes for the button: while `button` in styles.js has no specific position, we apply `position: absolute` here to achieve the desired design and therefore need to use `position`.